harvard-huit/Harvard-Geospatial-Library
# Adding Rake Task to delete guests.

**Adding Rake Task to delete guests.**
* * *


# What does this Pull Request do?
This adds a Rake Task to the application so that you can delete guests via direct sql queries.

# How should this be tested?

A description of what steps someone could take to:
* ssh into the container: `docker exec -it geoblacklight_web bash`
* Run the new rake command: `rake harvard_geospatial_library:delete_guests`
* You should see a success message telling you how many users, bookmarks and searches have been deleted.

# Test coverage
Yes/No: Are changes in this pull-request covered by:
- unit tests? No
- integration tests? No

# Interested parties
@enriquediaz @dl-maura
